resources/com/mesosphere/global/terraform_file_deploy.sh

```diff
@@ -100,20 +100,40 @@ EOF
   fi
   cat <<EOF > marathon-lb-options.json
 {
-    "marathon-lb": {
```

Comment:
we already have here all correct values, default from the package as we start a dcos `strict` cluster, it would break all integration tests if we change it.
we should stay with these minimal defaults from the `marathon-lb` options.

Reply:
fixed at https://github.com/OleksandrBielov/jenkins-library/commit/1bd4b9127b8d219de1e3ce02dd03ee08e36041f3